.github/assets/Dolphin-memory-engine.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Type=Application
+Name=Dolphin memory engine
```

Comment:
Dolphin Memory Engine for consistency

Reply:
Upper is better. I was matching the binary.

Reply:
I think I'll change the binary and repo to be all lower to match usual app naming and repo standards, but any displayed situation should be Dolphin Memory Engine I think, or DME. Whatever makes more sense.